Catalog.html

```diff
@@ -0,0 +1,362 @@
+          method="get"
+        >
+          <fieldset class="infrastructure-filter-group">
+            <legend class="infrastructure-filter-title">Инфраструктура</legend>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="swimming pool"
+              />
+              <span class="control-mark"></span>
+              <span class="control-label">Бассейн</span>
+            </label>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="parking space"
+              />
+              <span class="control-mark"></span>
+              <span class="control-label">Парковка</span>
+            </label>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="Wi-fi"
+              />
+              <span class="control-mark"></span>
+              <span class="control-label">Wi-fi</span>
+            </label>
+          </fieldset>
+          <fieldset class="housing-filter-group">
+            <legend class="housing-filter-title">Тип жилья</legend>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="hotel"
+              />
+              <span class="control-mark"></span>
+              <span class="control-label">Гостиница</span>
+            </label>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="motel"
+              />
+              <span class="control-mark"></span>
+              <span class="control-label">Мотель</span>
+            </label>
+            <label class="control">
+              <input
+                class="visually-hidden control-input"
+                type="checkbox"
+                name="apartments"
+              />
+              <span class="control-mark"></span>
+              <span class="control-label">Апартаменты</span>
+            </label>
+          </fieldset>
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Стоимость, ₽:</legend>
+            <button class="range-toggle toggle-min" type="button">
+              <span class="visually-hidden">Изменить минимальную цену.</span>
+            </button>
+            <button class="range-toggle toggle-max" type="button">
+              <span class="visually-hidden">Изменить максимальную цену.</span>
+            </button>
+            <label class="catalog-filter-label">
+              от
+              <input
+                class="catalog-filter-input"
+                type="number"
+                name="min-price"
+              />
```

Comment:
для второго так же, но с другим значением по умолчанию
```suggestion
              <input
                class="catalog-filter-input"
                type="number"
                name="min-price"
                value="0"
                min="0" 
                max="10000"
              />
```